src/main/views/course/index.html

```diff
@@ -8,29 +8,30 @@
 		<h2>Active Courses <i class="material-icons">help</i></h2>
 		<table>
 			<tr>
+				
 				<th>Course</th>
 				<th>Semester</th>
 				<th>Year</th>
 				<th>Artifact Progress <i class="material-icons">help</i></th>
 				<th>Due Date</th>
 				<th></th>
 			</tr>
+			{{#artifacts_active}}
```

Comment:
Indent to match next line?

Reply:
moved {{#artifacts}} (originally {{#arti}} ) to match level of indentation as {{#artifacts_active}}